detekt-config.yml

```
@@ -0,0 +1,615 @@
+build:
+  maxIssues: 3
```

```suggestion
  maxIssues: 0
```
It's better to fix all issues at the start.